mobile-app/app/screens/AppNavigator/screens/Settings/SettingsScreen.tsx

```diff
@@ -234,7 +236,8 @@ function PrivacyLockToggle ({
         style={tailwind('flex p-4 pr-2 flex-row items-center justify-between')}
       >
         <ThemedText testID='text_privacy_lock' style={tailwind('font-medium')}>
-          {translate('screens/Settings', 'Privacy Lock')}
+          {authenticationName !== undefined &&
+            `${translate('screens/Settings', 'Secure with')} ${translate('screens/Settings', authenticationName)}`}
```

Comment:
This should accept param. In case on other languages, the format is not the same. E.g, `Secure with {{option}}`

Reply:
updated